ecs/accounts/templates/home.html

```diff
@@ -1,11 +1,15 @@
 {% extends 'base.html' %}
 
 {% block title %}
+    {% load static from staticfiles %}
```

Comment:
In Django 1.10 and later, you can use {% load static %} instead.
Pomyliłem się, coś mi nie pasowało te staticfiles :P